### PR TITLE
Crud publisher

### PR DESCRIPTION
**Pull request by hoangtv2605**
## Related Tickets:
- [26957](https://edu-redmine.sun-asterisk.vn/issues/26957)

## WHAT this PR does?
- Show list, create, edit, delete publisher

## Notes Impacted Areas:
- N/A.

## Attachments:
layout
layout1
layout2

### CODE REVIEW

**Comment by hoangtv2605** [comment]
ready

File: app/Http/Controllers/PublisherController.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    public function create()
+    {
+        return view('admin.publisher.add');
+    }
+
+    public function store(PublisherRequest $request)
+    {
+        $publisher = $request->all();
+        Publisher::create($publisher);
```
Comment (hoangminhuet) [inline app/Http/Controllers/PublisherController.php]:
```suggestion
        Publisher::create($request->all());
```

File: app/Http/Controllers/PublisherController.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    public function update(Request $request, $id)
+    {
+        $publisher = Publisher::find($id);
+        $publisher->name = $request->name;
+        $publisher->email = $request->email;
+        $publisher->address = $request->address;
+
+        $publisher->save();
```
Comment (hoangminhuet) [inline app/Http/Controllers/PublisherController.php]: thử dùng `$publisher->update($request->all())` xem anh :v

File: resources/views/admin/publisher/index.blade.php [hunk 3]
```diff
@@ -6,6 +6,11 @@
 
 @section('content')
 <div class="content-wrapper">
+        @if (session()->get('success'))
+        <div class="alert alert-success alert-dismissible fade show" role="alert">
+            {{ session()->get('success') }}
+        </div>
+    @endif
```
Comment (hoangminhuet) [inline resources/views/admin/publisher/index.blade.php]: indent 4 spaces

File: routes/web.php [hunk 4]
```diff
@@ -16,3 +16,7 @@
 Route::get('/', function () {
     return view('welcome');
 });
```
Comment (hoangminhuet) [inline routes/web.php]: k viết function trong route anh :v

**Pull request merged**